macros/query_comment.sql

```diff
@@ -23,6 +23,7 @@
             node_resource_type=node.resource_type,
             node_meta=node.config.meta,
             node_tags=node.tags,
+            run_started_at=run_started_at.astimezone(modules.pytz.utc).isoformat(),
             full_refresh=flags.FULL_REFRESH,
```

Comment:
Should this go up higher with the `invocation_id` and other non-node specific details?